src/AppInstallerCLICore/ChannelStreams.cpp

```diff
@@ -100,4 +100,16 @@ namespace AppInstaller::CLI::Execution
         m_out << f;
         return *this;
     }
+
+    NullStream::NullStream()
+    {
+        nOut = new std::ostream(&nullStreamBuf);

```

Comment:
>nOut = new std::ostream(&nullStreamBuf); [](start = 8, length = 40)

This should just be a `std::ostream m_out` in the header, no need for a pointer (and if you were to need one, std::unique_ptr).